src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -127,7 +130,8 @@ const NetEmitterOptionsSchema: JSONSchemaType<NetEmitterOptions> = {
         "new-project": { type: "boolean", nullable: true },
         csharpGeneratorPath: { type: "string", nullable: true },
         "clear-output-folder": { type: "boolean", nullable: true },
-        "save-inputs": { type: "boolean", nullable: true }
+        "save-inputs": { type: "boolean", nullable: true },
+        "model-namespace": { type: "boolean", nullable: true }
```

Comment:
I think we also need the 'namespace' from cadl-project.yaml,   do we already get that value?

Reply:
yes